From 63efe539dc4ef624647a72dc745ad29740e50103 Mon Sep 17 00:00:00 2001
From: Howard Hsu <howard-yh.hsu@mediatek.com>
Date: Tue, 22 Apr 2025 10:48:10 +0800
Subject: [PATCH 82/95] mtk: mt76: mt7996: rework the init value for current tx
 duty cycle

The init value of current tx duty cycle is 0 because firmware will not
send the unsolicited event with current tx duty cycle every second. Add
the init value as 100 to corrent the inin tx duty cycle.

Signed-off-by: Howard Hsu <howard-yh.hsu@mediatek.com>
---
 mt7996/mcu.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/mt7996/mcu.c b/mt7996/mcu.c
index fca4bd6d..56537c8a 100644
--- a/mt7996/mcu.c
+++ b/mt7996/mcu.c
@@ -6310,6 +6310,8 @@ int mt7996_mcu_set_thermal_throttling(struct mt7996_phy *phy, u8 state)
 			return ret;
 	}
 
+	phy->throttle_state = state;
+
 	return 0;
 }
 
-- 
2.45.2

